docs/services/azure-services-guide.md

```diff
@@ -0,0 +1,625 @@
+  name: azure-mysql-standard
+  namespace: azure-infra-dev
+EOF
+```
+
+* Create the `MySQLInstanceClass`:
+```bash
+kubectl apply -f mysql-standard.yaml
+```
+
+* You should see the following output:
+```bash
+mysqlinstanceclass.database.crossplane.io/mysql-standard created
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get mysqlinstanceclasses -n app-project1-dev
+```
+*Output*
+```bash
+NAME             AGE
+mysql-standard   27s
+```
+
+Once again, you are free to create more `MySQLInstanceClass` instances in this
+namespace to define more classes of service. For instance, if you created
+`mysql-azure-large` above, you may want to create a `MySQLInstanceClass` named
+`mysql-large` that references it. You may also choose to create MySQL resource
+classes for other non-Azure providers, and reference them for a class of service
+in the `app-project1-dev` namespace.
+
+You may specify *one* instance of a portable class kind as *default* in each
+namespace. This means that the portable resource class instance will be applied
+to claims that do not directly reference a portable class. If we wanted to make
+our `mysql-standard` instance the default `MySQLInstanceClass` for namespace
+`app-project1-dev`, we could do so by adding a label:
+
+```yaml
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstanceClass
+metadata:
+  name: mysql-standard
+  namespace: app-project1-dev
+  labels:
+    default: "true"
+classRef:
+  kind: SQLServerClass
+  apiVersion: database.azure.crossplane.io/v1alpha2
+  name: azure-mysql-standard
+  namespace: azure-infra-dev
+```
+
+#### Resource Claims
+Resource claims are used to create external resources by referencing a class of
+service in the claim's namespace. When a claim is created, Crossplane uses the
+referenced portable class to find a cloud-speific resource class to use as the
+configuration for the external resource. We need a to create a claim to
+provision the MySQL database we will use with Azure.
+
+* Define a `MySQLInstance` claim in `mysql-claim.yaml`:
+```yaml
+cat > mysql-claim.yaml <<EOF
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstance
+metadata:
+  name: mysql-claim
+  namespace: app-project1-dev
+spec:
+  classRef:
+    name: mysql-standard
+  writeConnectionSecretToRef:
+    name: wordpressmysql
+  engineVersion: "5.6"
+EOF
+```
+
+* Create the `MySQLInstance`:
+```bash
+kubectl apply -f mysql-claim.yaml
+```
+
+What we are looking for is for `STATUS` value to become `Bound` which indicates
+the managed resource was successfully provisioned and is ready for consumption.
+You can see when claim is bound using the following:
+
+*Command*
+```bash
+kubectl get mysqlinstances -n app-project1-dev
+```
+
+*Output*
+```bash
+NAME          STATUS   CLASS            VERSION   AGE
+mysql-claim   Bound    mysql-standard   5.6       11m
+```
+
+If the `STATUS` is blank, we are still waiting for the claim to become bound.
```

Comment:
Can we use `--wait` with `kubectl` to have the command wait for us? It may make sense for us to tell the user how to do that, to show them how it can be done